jobs/promptLicenceCreationService.test.ts

```diff
@@ -82,15 +84,21 @@ describe('prompt licence creation service ', () => {
   })
 })
 
-function createManagedCase(offenderManagers: OffenderManager[], licenceStatus: LicenceStatus) {
+function createManagedCase(offenderManagers: OffenderManager[], licenceStatus: LicenceStatus): ManagedCase {
   return {
     deliusRecord: { offenderId: 1, offenderManagers },
+    cvlFields: {
+      licenceType: 'AP',
+      hardStopDate: '2023-01-03',
+      hardStopWarningDate: '2023-01-01',
```

Comment:
I'm right in thinking that this is straight from NOMIS, not the CVL DB?

Reply:
yes, it is - because you could have multiple licences and we don't have a standard way to prioritise which one to use.

The date format here is wrong - bits in the cvl fields are formatted like cvl dates! I'll fix this